packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -428,6 +430,14 @@ class GestureDetector extends StatelessWidget {
   ///  * [kTertiaryButton], the button this callback responds to.
   final GestureTapCancelCallback onTertiaryTapCancel;
 
+  /// A pointer that might cause a double tap has contacted the screen at a
+  /// particular location.
+  ///
+  /// See also:
+  ///
+  ///  * [kTertiaryButton], the button this callback responds to.
```

Comment:
`kTertiaryButton`? Should probably be primary, no?

Reply:
Good catch - fixed.